docs/api-specs/v2.json

```diff
@@ -5,6 +5,100 @@
     "version": "v2"
   },
   "paths": {
+    "/v2/teachers/{trn}/itt-outcome": {
+      "put": {
+        "tags": [
+          "IttOutcome"
+        ],
+        "summary": "Sets ITT outcome for a teacher",
+        "parameters": [
+          {
+            "name": "trn",
+            "in": "path",
+            "description": "The TRN of the teacher to set ITT outcome for.",
```

Comment:
@benilovj should we go with `trainee` at this point, rather than `teacher`?

Reply:
hmm actually, in the context of DQT - every record is a teacher even though we know that's not correct so let's leave as is.